packages/amplify-e2e-core/src/utils/nexpect.ts

```diff
@@ -380,22 +378,15 @@ function chain(context: Context): ExecutionContext {
     let stdout: string[] = [];
     let noOutputTimer;
 
-    let logDumpFile: fs.WriteStream;
+    let stdoutPipe: NodeJS.WriteStream;
 
     if (process.env.VERBOSE_LOGGING_DO_NOT_USE_IN_CI_OR_YOU_WILL_BE_FIRED) {
-      const logdir = join(os.tmpdir(), 'amplify_e2e_logs');
-      fs.ensureDirSync(logdir);
-      const filename = join(logdir, `amplify_e2e_log_${generateRandomShortId()}`);
-      logDumpFile = fs.createWriteStream(filename);
-      console.log(`CLI test logs at [${filename}]`);
+      stdoutPipe = process.stdout;
```

Comment:
Can we have both files and stdout?
I got used to grepping them, plus some of the outputs are really long, i.e. longer than console's buffer size.

Reply:
Would it work for you to run the test locally and redirect stdout to a file? ie `yarn e2e path/to/test -t 'some test' > test-log.txt`

Reply:
It wouldn't because I'd get all logs in single file. I do leverage fact that each command ends up logged in separate file.

Reply:
Also I usually sort logs by timestamp to more-less reflect event history. Then I check them one by one starting from tail. I.e.
![image](https://user-images.githubusercontent.com/5849952/228115468-24b9c581-5ad8-47f2-89f9-7aade5d9bc5b.png)